<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StayScreenedAndSafe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2fa">
      <attribute name="title">
       <string>2FA Doğrulama</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2fa">
       <item>
        <widget class="QLabel" name="username_label">
         <property name="text">
          <string>Kullanıcı Adı:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="username_entry"/>
       </item>
       <item>
        <widget class="QPushButton" name="generate_button">
         <property name="text">
          <string>QR Kod Oluştur/Göster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="qr_label"/>
       </item>
       <item>
        <widget class="QLabel" name="code_label">
         <property name="text">
          <string>2FA Kodu:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="code_entry"/>
       </item>
       <item>
        <widget class="QPushButton" name="verify_button">
         <property name="text">
          <string>Kodu Doğrula</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_face">
      <attribute name="title">
       <string>Yüz Tanıma</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_face">
       <item>
        <widget class="QPushButton" name="face_register_button">
         <property name="text">
          <string>Yüz Kayıt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="face_verify_button">
         <property name="text">
          <string>Yüz Doğrulama</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
